src/module/actor/sheet/item-summary-renderer.ts

```diff
@@ -195,7 +195,7 @@ export class CreatureSheetItemRenderer<AType extends CreaturePF2e> extends ItemS
                 }
 
                 if (actor instanceof CharacterPF2e) {
-                    if (chatData.variants) {
+                    if ((chatData.variants as Array<unknown>).length) {
```

Comment:
Is that always going to be present for all items?

Reply:
Currently yes. Alternatively I can set it to undefined if no variants exist and leave the check as it is.

Reply:
That sounds better

Reply:
Done